src/components/Activities/Activity3/Game/TimeAllocation.tsx

```diff
@@ -118,34 +118,36 @@ function TimeAllocation(props: TimeAllocationProps): JSX.Element {
   };
 
   return <div id={'time-container'} className={tutorialStage < TUTORIAL_END ? 'enable-blur' : ''}>
-    <div id={'time-tutorial-overlay'} style={{display: `${tutorialStage >= TUTORIAL_END ? 'none' : ''}`}}/>
-    {
-      Object.values([0, 1, 2]).map((index) => {
-        return (
-          <div key={index} id={'time-tutorial-bubble'} className={tutorialStage === index ? 'disable-blur' : ''}
-            style={{
-              display: `${(tutorialStage === index) ? '' : 'none'}`,
-              alignSelf: 'flex-center',
-            }}>
-            {displayTutorialText()}
-            <button className='playnet-button' style={{zIndex: 50}} onClick={advanceTutorial}>Continue</button>
-          </div>
-        );
-      })
-    }
+    <div id={'time-tutorial-overlay'} className={'disable-blur'} style={{display: `${tutorialStage >= TUTORIAL_END ? 'none' : ''}`}}>
```

Comment:
display none is a lil funny, lets use JSX~
```suggestion
    {
     isTutorial && <div id={'time-tutorial-overlay'} className={'disable-blur'}>```